nixos/modules/services/misc/paperless.nix

```
@@ -335,7 +346,9 @@ in
       environment = env;
 
       preStart = ''
-        ln -sf ${manage} ${cfg.dataDir}/paperless-manage
+        # remove old papaerless-manage symlink
+        # TODO: drop with NixOS 25.11
+        [[ -L ${cfg.dataDir}/paperless-manage ]] && rm ${cfg.dataDir}/paperless-manage
```

```suggestion
        [[ -L '${cfg.dataDir}/paperless-manage' ]] && rm '${cfg.dataDir}/paperless-manage'
```
`dataDir` can contain spaces.